azure-mgmt-documentdb/src/main/java/com/microsoft/azure/management/documentdb/DatabaseAccount.java

```diff
@@ -59,25 +58,30 @@
     ConsistencyPolicy consistencyPolicy();
 
     /**
-     * @return an array that contains the regions ordered by their failover priorities
+     * @return the default consistency level for the DocumentDB database account
      */
-    Map<String, FailoverPolicyInner> failoverPolicies();
+    DefaultConsistencyLevel defaultConsistencyLevel();
 
     /**
-     * @return an array that contains the georeplication locations enabled for the DocumentDB account
+     * @return an array that contains the writable georeplication locations enabled for the DocumentDB account
      */
-    Map<String, Location> locations();
+    List<Location> writableReplications();
+
+    /**
+     * @return an array that contains the readable georeplication locations enabled for the DocumentDB account
+     */
+    List<Location> readableReplications();
 
     /**
      * @param failoverPolicies the failover policies
      */
-    void failoverPriorityChange(FailoverPolicies failoverPolicies);
+    void failoverPriorityChange(List<Location> failoverPolicies);

```

Comment:
Can this be `Collection<Location>` rather than `List<>`?  (Is the order meaningful?)